### ISSUE TITLE
[Bug]: Mobile: Account ledger jumps on first selection

### ISSUE THREAD

**Issue by shall0pass** [actualbudget/actual#1621]
### Verified issue does not already exist?

- [X] I have searched and found no existing issue

### What happened?

If I scroll down an account ledger on the mobile interface and select a transaction, the first time I try the interface will jump instead of opening the transaction.  The second attempt is  successful. The video below only shows the first attempt on the demo budget.

https://github.com/actualbudget/actual/assets/20625555/13f0fd77-ca41-40e1-93a5-495f917547c2

### Where are you hosting Actual?

Other

### What browsers are you seeing the problem on?

Chrome

### Operating System

Mobile Device

**Comment by Kidglove57** [comment]
I am running a recent Edge version on iOS 16. For me, selected transactions are opening immediately. I attach a brief screen recording.
https://github.com/actualbudget/actual/assets/93882449/d68e230b-cde8-40f9-884c-8685f9a305c7

**Comment by shall0pass** [comment]
Hmm. I see it with both Brave and Firefox on Android.

**Comment by Kidglove57** [comment]
Sorry to be the "spoilsport" here! I have tested on iOS in Safari, Chrome, Brave and Firefox but have not been able to replicate this.

My "Edge" version is about 10 days old - if that makes any difference?

**Comment by shall0pass** [comment]
😄 could you try on the demo budget online? My Edge version is only 2 days old and my video was on the demo version which is current with the master branch.

**Comment by shall0pass** [comment]
Maybe it's a recently introduced bug.  I can replicate this on Windows 11 too.
![jumping_account_view](https://github.com/actualbudget/actual/assets/20625555/d10b524c-4dfa-496c-bd7f-6cb079fce31f)

**Comment by shall0pass** [comment]
Odd. Try scrolling down a bit first.

**Comment by Kidglove57** [comment]
This is interesting:
1) I am still unable to reproduce this in my production file.
2) Using the Demo file - and scrolling down further as requested - I am now sometimes able to reproduce what you are seeing. But not consistently. 

If I do reproduce this it is only on the first click in the transaction register and all works fine after that. It feels just like it would if I did not clicked cleanly in the required place. Like I had "miss hit". Without your perseverance I think I would just have passed it off as a passing glitch.

**Comment by joel-jeremy** [comment]
I am able to reproduce this on the demo instance.

https://github.com/actualbudget/actual/assets/20313680/2d66f446-e9f4-45d0-8800-0b93a29c6fb9

**Comment by MatissJanis** [comment]
Repro:

1. open a single account on mobile
2. scroll down
3. click BETWEEN transactions on the right side (so right of the date)
4. scroll will jump to top

I am not sure what's going on here.. but good catch!

**Comment by trevdor** [comment]
Turns out the first time you interact with the list, react-aria sets the focused element to the first list item, then scrolls that focused element into view. 
I think this was a bug, and the relevant code has changed pretty dramatically in the latest version of `useSelectableList`. 

Appears fixed by upgrading. I went ahead and upgrade all react-aria packages, as they get frequent updates and work with one another. I'll do some thorough manual testing of the mobile components that use these libs.
